### PR TITLE
RM-38406 update libxmljs version

### PR DESCRIPTION
**Pull request by kin-lee-ELH**
update libxmljs to latest version

### CODE REVIEW

File: CODEOWNERS
Comment (ronaldpedagat) [inline CODEOWNERS:1]: 🚀

File: package-lock.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "blue-button-xml",
-  "version": "1.8.4",
+  "version": "1.8.1",
```
Comment (ronaldpedagat) [inline package-lock.json]: we're going backwards? 😆

File: lib/xml.js [hunk 2]
```diff
@@ -4,10 +4,10 @@ var libxmljs = require('libxmljs');
 
 exports.parse = function (src) {
   var options = {
-    noblanks: true
+    preserveWhitespace: false
   };
 
-  return libxmljs.parseXmlString(src, options);
+  return libxmljs.parseXml(src, options);
```
Comment (ronaldpedagat) [inline lib/xml.js:10]: did we measure performance differences? between `parseXmlString` vs `parseXml`

**Comment by ronaldpedagat** [comment]
Would it be better to attach this PR to https://elationhealth.atlassian.net/browse/RM-38779 ?

File: lib/xml.js (hunk 2 above)
Comment (kin-lee-ELH) [reply lib/xml.js:10]: `parseXmlString` deprecated in latest version

**Review by ronaldpedagat — APPROVED** [review]

**Pull request merged**